lib/gap_intelligence/client/merchant_pricing_trend_downloads.rb

```diff
@@ -0,0 +1,18 @@
+module GapIntelligence
+  # @see http://api.gapintelligence.com/api/doc/v1/merchant_pricing_trend_downloads.html
+  module MerchantPricingTrendDownloads
+    # Creates new merchant pricing trend download
+    #
+    # @param owner_id [String,Integer] owner id of the download
+    # @param params [Hash] parameters that will be the JSON payload of the http request
+    # @param options [Hash] options the options to make the request with
+    # @yield [request] The Faraday request
+    # @return [Download] the created download
+    # @return [RequestError] error messages
+    # @see http://api.gapintelligence.com/api/doc/v1/merchant_pricing_trend_downloads/create.html
+    def create_merchant_pricing_trend_download(owner_id, params, options = {}, &block)
+      default_option(options, :record_class, MerchantPricingTrendDownload)
+      perform_request(:post, 'merchant_pricing_trend_downloads', options.merge(body: params.merge({ owner_id: owner_id })), &block)
```

Comment:
probably a question to @GapIntelligence/dev-team, not only to @gtimti - in Download (not MPTDownload) we send all download params under `download` key, here we send MPT params as they are, they are not under one `merchant_pricing_trend_download` key (or whatever the name should be). Do we need that to be consistent?